doc/changelog.rst

```diff
@@ -257,8 +257,8 @@ Breaking Changes in 4.0
 - The "tls" install extra is no longer necessary or supported and will be
   ignored by pip.
 - ``tz_aware``, an argument for :class:`~bson.json_util.JSONOptions`,
-  now defaults to ``False`` instead of ``True``. ``json_util.loads`` now
-  decodes datetime as naive by default.
+  now defaults to ``False`` instead of ``True``. :meth:`bson.json_util.loads` now
+  decodes datetime as naive by default. See :ref:`tz_aware_default_change` for more info.
```

Comment:
It might be worth saying something like "...to match the default behavior of bson.decode".